src/crypto/OlmDevice.js

```diff
@@ -158,21 +175,35 @@ OlmDevice.getOlmVersion = function() {
 
 
 /**
- * extract our OlmAccount from the session store and call the given function
+ * extract our OlmAccount from the crypto store and call the given function
+ * with the account object and a 'save' function which returns a promise.
+ * The function will not be awaited upon and the save function must be
```

Comment:
this is maybe a bit misleading. The function *will* be awaited upon, by virtue of any promise it returns being returned by this `async` function - so the promise returned by `func` must resolve before the promise returned by `_getAccount` will resolve.

It would probably be clearer to say "The `account` will be freed as soon as `func` returns - even if `func` returns a promise".

Reply:
yeah, agreed. done.